traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1623,9 +1623,7 @@ sub check_lwp_response_content_length {
 	my $url           = $lwp_response->request->uri;
 
 	if ( !defined($lwp_response->header('Content-Length')) ) {
-		( $log_level >> $panic_level ) && print $log_level_str . " $url did not return a Content-Length header!\n";
-		exit;
-		return 1;
+		return 0; # Content-Length MAY be omitted per HTTP/1.1 RFC 7230, and in fact MUST NOT be included with a 'Transfer-Encoding: Chunked' header, which MUST be accepted by clients.
```

Comment:
X prefixes are deprecated https://tools.ietf.org/html/rfc6648

Further, X-prefixed or not, it doesn't solve the problem the deprecation addressed, of allowing inconsistent adoption. https://trac.ietf.org/trac/httpbis/ticket/178

How about `Whole-Content-SHA512`?